share/Makefile

```diff
@@ -3,7 +3,7 @@
 .PHONY: all dist touch-po update-po extract-pot
 
 DISTNAME = Zonemaster-CLI
-POFILES != echo *.po
+POFILES != find . -type f -name '*.po' -depth 1
```

Comment:
I think you meant `-maxdepth 1`

Reply:
No, I meant `-depth 1`, but when the value is 1 both give the same result. If you see any problem with `-depth` that `-maxdepth` does not have I can change.

And as you commented in zonemaster/zonemaster-engine/pull/804, we do not expect any other PO files in that subtree so with and without `-(max)depth 1` will give that same result in practice.

Reply:
I see that `-depth n` does not seem to be in Linux find(). Changed to `-maxdepth 1`. And it also wants that to be before the file pattern. See update.